charts/metering-ansible-operator/templates/crds/prestotable.crd.yaml

```diff
@@ -21,3 +21,105 @@ spec:
   - name: Table Name
     type: string
     JSONPath: .status.tableName
+  validation:
+    openAPIV3Schema:
+      description: 'PrestoTable is a custom resource that represents a database table accessible from within Presto.
```

Comment:
Use `: |` and remove the quotes, and indent this at the next level on a newline for a multiline string. https://yaml-multiline.info/

Reply:
Do this for all descriptions.